CHANGELOG.md

```diff
@@ -4,6 +4,103 @@ All notable changes to this project will be documented in this file.
+ - JSON-RPC: elements network PSET now only supports PSETv2. ([#5898])
+ - JSON-RPC: `close`, `fundchannel`, `fundpsbt`, `multifundchannel`, `multiwithdraw`, `txprepare`, `upgradewallet`, `withdraw` `feerate` (`feerange` for `close`) value *slow* is now 100 block-estimate, not half of 100-block estimate. ([#6120])
+ - Protocol: spending unilateral close transactions now use dynamic fees based on deadlines (and RBF), instead of fixed fees. ([#6120])
+ - Protocol: Allow slight overpaying, even with MPP, as spec now recommends. ([#6138])
+ - `msggen`: The generated interfaces `cln-rpc` anc `cln-grpc` can now work with a range of versions rather than having to match the CLN version ([#6142])
+ - `grpc`: The mTLS private keys are no longer group-readable ([#6075])
+
+### Deprecated
+
+Note: You should always set `allow-deprecated-apis=false` to test for changes.
+
+ - JSON-RPC: `close`, `fundchannel`, `fundpsbt`, `multifundchannel`, `multiwithdraw`, `txprepare`, `upgradewallet`, `withdraw` `feerate` (`feerange` for `close`) expressed as, "delayed_to_us", "htlc_resolution", "max_acceptable" or "min_acceptable".  Use explicit block counts or *slow*/*normal*/*urgent*/*minimum*. ([#6120])
+ - Plugins: `estimatefees` returning feerates by name (e.g. "opening"); use `fee_floor` and `feerates`. ([#6120])
+ - Protocol: Not setting `option_scid_alias` in `option_channel` `channel_type` for unannounced channels. ([#6136])
+ 
+
+### Removed
+
+ - JSON-RPC: the "msat" suffix on millisatoshi fields, as deprecated in v0.12.0. ([#5986])
+ - JSON-RPC: all the non-msat-named millisatoshi fields deprecated in v0.12.0. ([#5986])
+ - JSON-RPC: `listpeers`.`local_msat` and `listpeers`.`remote_msat` (deprecated v0.12.0) ([#5986])
+ - JSON-RPC: `checkmessage` now always returns an error when the pubkey is not specified and it is unknown in the network graph (deprecated v0.12.0) ([#5986])
+ - JSON-RPC: require the `"jsonrpc": "2.0"` property (requests without this deprecated in v0.10.2). ([#5986])
+
+
+### Fixed
+
+ - Plugins: `bcli` now tells us the minimal possible feerate, such as with mempool congestion, rather than assuming 1 sat/vbyte. ([#6120])
+ - `lightningd`: don't log gratuitous "Peer transient failure" message on first connection after restart. ([#6140])
+ - `channeld`: no longer spin and spam logs when waiting for revoke_and_ack. ([#6107])
+ - Plugin: `autoclean` now also cleans forwards with status `local_failed` ([#6109])
+ - Protocol: we will upfront reject channel_open which asks for a zeroconf channel unless we are going to do a zerconf channel. ([#6136])
+ - Protocol: We now correctly accept the `option_scid_alias` bit in `open_channel` `channel_type`. ([#6136])
+ - JSON-RPC: `feerates` document correctly that urgent means 6 blocks (not 2), and give better feerate examples. ([#6170])
+ - `wallet`: we no longer make txs below minrelaytxfee or mempoolminfee. ([#6073])
+ - `delpay`: be more pedantic about delete logic by allowing ([#6115])
```

Comment:
Missed that last part of this changelog entry:
Changelog-Fixed: delpay be more pedantic about delete logic by allowing delete payments by status directly on the database.